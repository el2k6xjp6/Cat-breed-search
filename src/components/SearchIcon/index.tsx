import React from "react";
import { Svg } from "./styles";

function SearchIcon() {
  return (
    <Svg>
      <circle
        cx="12"
        cy="12"
        r="8"
        stroke="black"
        stroke-width="3"
        fill="white"
      />
      <line x1="18" y1="18" x2="26" y2="26" stroke="black" stroke-width="3px" />
    </Svg>
  );
}

export default React.memo(SearchIcon);
