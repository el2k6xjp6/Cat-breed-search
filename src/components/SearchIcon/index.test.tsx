import SearchIcon from ".";
import "jest-styled-components";
import renderer from "react-test-renderer";

describe("SearchIcon", () => {
  test("snapshot for SearchIcon", () => {
    const tree = renderer.create(<SearchIcon />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
