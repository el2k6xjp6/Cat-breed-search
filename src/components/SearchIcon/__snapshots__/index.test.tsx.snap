// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchIcon snapshot for SearchIcon 1`] = `
.c0 {
  height: 30px;
  width: 30px;
  margin-left: 10px;
}

<svg
  className="c0"
>
  <circle
    cx="12"
    cy="12"
    fill="white"
    r="8"
    stroke="black"
    strokeWidth="3"
  />
  <line
    stroke="black"
    strokeWidth="3px"
    x1="18"
    x2="26"
    y1="18"
    y2="26"
  />
</svg>
`;
